src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,211 @@
+package Duke;
+import Duke.todoList.task.listTypes;
+import Duke.todoList.todoList;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Duke {
+
+    public static void dukeStartup(){
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n" );
+
+    }
+
+    public static void dukeResponse(String output){
+        System.out.println("____________________________________________________________\n" +
+                output + "\n" +
+                "____________________________________________________________\n");
+
+    }
+
+    public static String command_parser(todoList list, String command){
```

Comment:
This method seems to be long to read. Perhaps abstracting away some details can help make the execution path more prominent. For suggestions see: [https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent)